libraries/AP_Scripting/lua_scripts.h

```diff
@@ -37,6 +37,9 @@ class lua_scripts
     void run(void);
 
     static bool overtime; // script exceeded it's execution slot, and we are bailing out
+
+    static void *_heap;
```

Comment:
Don't expose this, it leaks the abstraction and begs for bad/racy manipulation. Instead use `lua_getallocf` where you need it, and call the allocator that was returned to you.